avs_core/include/avs/capi.h

```diff
@@ -94,16 +94,24 @@
 
 #ifdef BUILDING_AVSCORE
 #ifdef AVS_WINDOWS
-#  define AVSC_EXPORT __declspec(dllexport)
+#  ifndef AVS_STATIC_LIB
+#    define AVSC_EXPORT __declspec(dllexport)
+#  else
+#    define AVSC_EXPORT
+#  endif
 #  define AVSC_API(ret, name) EXTERN_C AVSC_EXPORT ret AVSC_CC name
 #else
 #  define AVSC_EXPORT EXTERN_C
 #  define AVSC_API(ret, name) EXTERN_C ret AVSC_CC name
 #endif
 #else
-#  define AVSC_EXPORT EXTERN_C __declspec(dllexport)
```

Comment:
I thought this line was a mistake, since this is in the `AVS_WINDOWS` not defined branch, and it defines `AVSC_EXPORT` with a Windows-specific attribute. So I boldly changed it.

Reply:
It's not in the `AVS_WINDOWS` not defined branch, it's in the `BUILDING_AVSCORE` not defined branch.  And removing it breaks using the C interface on Linux.

Reply:
Sure, I can put it back. But why would C interface on Linux need `__declspec(dllexport)`?

Reply:
It doesn't, [__declspec is defined away to nothing when used on non-Windows](https://github.com/AviSynth/AviSynthPlus/blob/master/avs_core/include/avs/posix.h#L45), but the lack of the AVSC_EXPORT structure causes it to fail.

Reply:
Anyway, I reverted the line.